CHANGELOG.md

```diff
@@ -1,3 +1,36 @@
+# v18.0.0-snapshot
+
+Kubernetes API Version: 1.18.12
+
+**Important Information:**
+
+- The Kubernetes Python client versioning scheme has changed. The version numbers used till Kubernetes Python Client v12.y.z lagged behind the actual Kubernetes minor version numbers. From this release, the client is moving a version format `vY.Z.P` where `Y` and `Z` are respectively from the Kubernetes version `v1.Y.Z` and `P` would incremented due to changes on the Python client side itself. Ref: https://github.com/kubernetes-client/python/issues/1244
+- Python 2 had reached [End of Life](https://www.python.org/doc/sunset-python-2/) on January 1, 2020. The Kubernetes Python Client has dropped support for Python 2 from this release (v18.0.0) and will no longer provide support to older clients as per the [Kubernetes support policy](https://kubernetes.io/docs/setup/release/version-skew-policy/#supported-versions).
+
+**API Change:**
```

Comment:
Some major API change in "Generate client 18.0.0-snapshot":
```
kubernetes/client/api/apps_v1beta1_api.py                                          |  5658 ---
kubernetes/client/api/apps_v1beta2_api.py                                          |  9234 -----
kubernetes/client/api/extensions_v1beta1_api.py                                    |  9411 +----
```
Could you add a note about these APIs being removed from this client? 
```
- the following deprecated APIs can no longer be served:
  - All resources under `apps/v1beta1` and `apps/v1beta2` - use `apps/v1` instead
  - `daemonsets`, `deployments`, `replicasets` resources under `extensions/v1beta1` - use `apps/v1` instead
  - `networkpolicies` resources under `extensions/v1beta1` - use `networking.k8s.io/v1` instead
  - `podsecuritypolicies` resources under `extensions/v1beta1` - use `policy/v1beta1` instead ([#85903](https://github.com/kubernetes/kubernetes/pull/85903), [@liggitt](https://github.com/liggitt)) [SIG API Machinery, Apps, Cluster Lifecycle, Instrumentation and Testing]
```

Reply:
Thanks for reminding me. I had forgotten to put the removal notices.